inc/razorpay/class-camptix-payment-method-razorpay.php

```diff
@@ -104,6 +104,26 @@ public function camptix_init() {
 	 * @access public
 	 */
 	public function enqueue() {
+
+		if ( ! isset( $_GET['tix_action'] ) || ( 'attendee_info' !== $_GET['tix_action'] ) ) {

```

Comment:
@sanyogs This code is already in `master` branch. Did you reformat code or anything else
For ref: https://github.com/wpindiaorg/camptix-indian-payments/blob/master/campt-indian-payment-gateway.php#L177

Reply:
I have move required js file for razorpay under razor pay
and common file which is for both validation put in master